cns/service.go

```diff
@@ -25,15 +25,16 @@ import (
 
 const (
 	// Default CNS server URL.
-	defaultAPIServerURL = "tcp://localhost:10090"
-	genericData         = "com.microsoft.azure.network.generic"
+	defaultAPIServerURL  = "tcp://localhost:10090"
+	defaultAPIServerPort = "10090"
+	genericData          = "com.microsoft.azure.network.generic"
 )
 
 // Service defines Container Networking Service.
 type Service struct {
 	*common.Service
 	EndpointType string
-	Listener     *acn.Listener
+	Listeners    *[]acn.Listener
```

Comment:
this should be `[]*acn.Listener`. We want a list of Listener's pointers, not a pointer to a list of listeners.